### PR TITLE
Update the Ukrainian Translation

### PR DESCRIPTION
**Pull request by ZaHaDumGoth**
New batch of translated text.

### CODE REVIEW

File: files/lang/uk.po [hunk 1]
```diff
@@ -2664,19 +2666,19 @@ msgid "Exit Town"
...
 msgid "View Hero"
-msgstr ""
+msgstr "Оглянути героя"
 
 msgid "The above spells are available here."
 msgstr ""
 
 msgid "The spells the hero can learn have been added to their book."
-msgstr ""
+msgstr "Заклинання, які герой може вивчити, були додані в Книгу Заклинань."
```
[the remark this answers is not in the document]
Comment (ZaHaDumGoth) [reply files/lang/uk.po:2678]: I see. I can make this sentense shorted and without meaning loss.

File: files/lang/uk.po [hunk 2]
```diff
@@ -3080,139 +3103,149 @@ msgid ""
...
-msgstr ""
+msgstr "Відобразити додаткову інформацію як FPS та поточний час."
 
 msgid "Hotkey: "
-msgstr ""
+msgstr "Гаряча клавіша: "
 
 msgid "Category: "
-msgstr ""
+msgstr "Категорія:"
```
[the remark this answers is not in the document]
Comment (ZaHaDumGoth) [reply files/lang/uk.po:3112]: Clear.

File: files/lang/uk.po [hunk 3]
```diff
@@ -6490,7 +6567,7 @@ msgid ""
 msgstr ""
 
 msgid " and "
-msgstr ""
+msgstr "і"
```
[the remark this answers is not in the document]
Comment (ZaHaDumGoth) [reply files/lang/uk.po:6577]: Clear. Will add spaces before and after.

File: files/lang/uk.po [hunk 4]
```diff
@@ -2902,28 +2923,28 @@ msgid "3D Audio"
...
 msgid "Resource cost:"
-msgstr ""
+msgstr "Вартість ресурсу:"
 
 msgid "Total: "
 msgstr "Загалом: "
 
 msgid "Need: "
-msgstr ""
+msgstr "Потрібно:"
```
Comment (ihhub) [inline files/lang/uk.po:2947]: Please add an extra space after :

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @ZaHaDumGoth , I put 2 comments here. Can you please take a look at them?

File: files/lang/uk.po [hunk 5]
```diff
@@ -2956,19 +2977,19 @@ msgid "Language"
...
-msgstr ""
+msgstr "Чорно-біле"
 
 msgid "Mouse Cursor"
-msgstr ""
+msgstr "Вказівник миші"
 
 msgid "Color"
-msgstr ""
+msgstr "Колір"
```
[the remark this answers is not in the document]
Comment (ZaHaDumGoth) [reply files/lang/uk.po:2989]: I got what you mean and where is it in game. Its in Interface Preferences, right? Mean cursor is colored or black ?

File: files/lang/uk.po (hunk 4 above)
Comment (ZaHaDumGoth) [reply files/lang/uk.po]: Added.

File: files/lang/uk.po (hunk 5 above)
[the remark this answers is not in the document]
Comment (ZaHaDumGoth) [reply files/lang/uk.po]: Fixed.

[the remark this answers is not in the document]
Comment (ihhub) [reply files/lang/uk.po]: Yes, it was related to color or black and white cursor.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@ZaHaDumGoth , thank you so much for these updates!

**Pull request merged**